GCC 6.2 build fails - Mac OS X 10.5.8 PPC

I tried `brew install gcc6` and it got quite far, but failed as some point:

```
$ brew install gcc6
==> Installing dependencies for gcc6: gmp, mpfr, libmpc, isl014
==> Installing gcc6 dependency: gmp
==> Downloading https://ia902307.us.archive.org/31/items/tigerbrew/gmp-6.0.0a.le
######################################################################## 100.0%
==> Pouring gmp-6.0.0a.leopard_altivec.bottle.tar.gz
/usr/local/Cellar/gmp/6.0.0a: 15 files, 3.1M
==> Installing gcc6 dependency: mpfr
==> Downloading https://ia902307.us.archive.org/31/items/tigerbrew/mpfr-3.1.3.le
######################################################################## 100.0%
==> Pouring mpfr-3.1.3.leopard_altivec.bottle.tar.gz
/usr/local/Cellar/mpfr/3.1.3: 24 files, 3.5M
==> Installing gcc6 dependency: libmpc
==> Downloading https://ia902307.us.archive.org/31/items/tigerbrew/libmpc-1.0.3.
######################################################################## 100.0%
==> Pouring libmpc-1.0.3.leopard_altivec.bottle.tar.gz
/usr/local/Cellar/libmpc/1.0.3: 10 files, 352K
==> Installing gcc6 dependency: isl014
==> Downloading http://isl.gforge.inria.fr/isl-0.14.tar.bz2
######################################################################## 100.0%
==> ./configure --disable-silent-rules --prefix=/usr/local/Cellar/isl014/0.14 --
==> make
==> make install
==> Caveats
This formula is keg-only, which means it was not symlinked into /usr/local.

Conflicts with isl in main repository.

Generally there are no consequences of this for you. If you build your
own software and it requires this formula, you'll need to add to your
build variables:

    LDFLAGS:  -L/usr/local/opt/isl014/lib
    CPPFLAGS: -I/usr/local/opt/isl014/include

==> Summary
/usr/local/Cellar/isl014/0.14: 68 files, 3.0M, built in 7.9 minutes
==> Installing gcc6
==> Downloading https://ftpmirror.gnu.org/gcc/gcc-6.2.0/gcc-6.2.0.tar.bz2
######################################################################## 100.0%
==> Patching
patching file gcc/jit/Make-lang.in
==> ../configure --build=powerpc-apple-darwin9.8.0 --prefix=/usr/local/Cellar/gc
==> make bootstrap
Last 15 lines from /Users/oboroc/Library/Logs/Homebrew/gcc6/02.make:
checking sys/sdt.h in the target C library... yes
checking dl_iterate_phdr in target C library... unknown
checking whether to enable maintainer-specific portions of Makefiles... no
checking whether to avoid linking multiple front-ends at once... no
Links are now set up to build a native compiler for powerpc-apple-darwin9.8.0.
checking Checking for isl_options_set_schedule_serialize_sccs... no
checking for exported symbols... yes
checking for library containing dlopen... none required
checking for -fPIC -shared... no
configure: error: 
   Building GCC with plugin support requires a host that supports
   -fPIC, -shared, -ldl and -rdynamic.
make[2]: *** [configure-stage1-gcc] Error 1
make[1]: *** [stage1-bubble] Error 2
make: *** [bootstrap] Error 2

READ THIS: https://github.com/mistydemeo/tigerbrew/wiki/troubleshooting

```

--- a/Library/Formula/gcc.rb
+++ b/Library/Formula/gcc.rb
@@ -116,7 +116,7 @@ class Gcc < Formula
 
     # "Building GCC with plugin support requires a host that supports
     # -fPIC, -shared, -ldl and -rdynamic."
-    args << "--enable-plugin" if MacOS.version > :tiger
+    args << "--enable-plugin" if MacOS.version > :leopard
 
     # Otherwise make fails during comparison at stage 3
     # See: http://gcc.gnu.org/bugzilla/show_bug.cgi?id=45248

--- a/Library/Formula/gcc6.rb
+++ b/Library/Formula/gcc6.rb
@@ -123,7 +123,7 @@ class Gcc6 < Formula
 
     # "Building GCC with plugin support requires a host that supports
     # -fPIC, -shared, -ldl and -rdynamic."
-    args << "--enable-plugin" if MacOS.version > :tiger
+    args << "--enable-plugin" if MacOS.version > :leopard
 
     # The pre-Mavericks toolchain requires the older DWARF-2 debugging data
     # format to avoid failure during the stage 3 comparison of object files.

